app/views/govuk_publishing_components/audit/_components.html.erb

```diff
@@ -66,7 +66,26 @@
                 <%= component[:component] %> (<%= component[:count] %>)
               </dt>
               <dd class="govuk-summary-list__value">
-                <%= component[:list] %>
+                <p><%= component[:list] %></p>
+                <% component[:locations].each do |application| %>
+                  <% github_link = 'https://github.com/alphagov/' + application[:name] + '/blob/main/' %>
+                  <details class="govuk-details govuk-!-margin-bottom-2" data-module="govuk-details">
+                    <summary class="govuk-details__summary">
+                      <span class="govuk-details__summary-text">
+                        <%= application[:name] %> (<%= application[:locations].length %>)
```

Comment:
Just a suggestion to make it clearer what the numbers mean:

```suggestion
                        <%= application[:name] %> (<%= application[:locations].length %> uses)
```

It should hopefully make it clearer the difference between a number next to the component and the number next to the application.